api/v1/version.go

```diff
@@ -56,6 +56,8 @@ const (
 	VcluseropsAsDefaultDeploymentMethodMinVersion = "v24.1.0"
 	// Starting in v24.1.0, we use server logrotate and not depend on cron job
 	InDatabaseLogRotateMinVersion = "v24.1.0"
+	// Starting in v24.2.0 (not including), restoring from a restore point in archive is supported.
```

Comment:
We will support restore in 24.2.0.

Reply:
Isn't it true that
> 24.2.0 will be blocked too albeit temporarily.

?

Reply:
I copied this from JIRA ticket description. As discussed, we will now set v24.2.0 as the min version that supports restore.